src/module/documents/UTSCombat.mjs

```diff
@@ -9,4 +12,27 @@ export default class UTSCombat extends Combat {
      */
     Hooks.callAll("UTS.prepareCombatData", this);
   }
+
+  /**
+   * Adds a player combatant to the current combat
+   */
+  async addPlayer() {
+    const data = {
+      type: "player",
+      system: {}
+    };
+    data.system.user = await foundry.applications.api.DialogV2.prompt({
+      window: {title: game.i18n.localize("UTS.Combat.AddPlayerCombatTracker")},
+      content: Player.schema.getField("user").toFormGroup().outerHTML,
+      rejectClose: true,
+      ok: {
+        callback: (event, button, dialog) => button.form.elements["system.user"].value
+      }
+    });
+    const user = game.users.get(data.system.user);
+    if (!user) return;
+    data.name = user.name;
+    data.img = user.avatar;
+    await this.createEmbeddedDocuments("Combatant", [data]);
```

Comment:
Since this is API, we should give it a proper return value (of just 1 combatant).